### PR TITLE
Added script for coverage PoliciesSecurity RAI_Default_Policies_forAP…

### PR DESCRIPTION
**Pull request by Aleksandriys**
@aarotar 
Please review.

### CODE REVIEW

File: test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+  self.mobileSession2:ExpectNotification("OnPermissionsChange")
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test Case")
+--Copy prepared JSON file with ALLOWED RPC ListFiles, in /tmp/fs/mp/images/ivsu_cache/ - and use it.
+--Example: os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/PTU_NewPolicy_GENEVI.json /tmp/fs/mp/images/ivsu_cache/") 
+function Test:AssignNewDefaultPoicies()
+   policyUpdate(self, "/tmp/fs/mp/images/ivsu_cache/PTU_NewPolicy_GENEVI.json")
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua]: @Aleksandriys , path to PTU file should not be dependent on you local machine

File: test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        pathToSnaphot
+      )
+      EXPECT_HMICALL("BasicCommunication.SystemRequest")
+      :Do(function(_,data)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end)
+      EXPECT_RESPONSE(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+      :Do(function(_,data)
+          --Copy of JSON file in /tmp/fs/mp/images/ivsu_cache/
+          --Example: os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/ptu.json /tmp/fs/mp/images/ivsu_cache/")
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua]: @Aleksandriys , remove commented code

File: test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua (hunk 1 above)
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua]: @Aleksandriys , file PTU_NewPolicy_GENEVI.json is not committed - so I cannot check correctness of script bahavior

**Comment by Aleksandriys** [comment]
@maria-mih please review

**Comment by Aleksandriys** [comment]
Please have a look of thie request:      
https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/181

**Pull request closed without merging**